specifyweb/stored_queries/format.py

```diff
@@ -236,7 +235,7 @@ def aggregate(self, query: QueryConstruct,
         aggregated = blank_nulls(group_concat(formatted, separator, *order_by_expr))
 
 
-        return subquery.query.add_column(aggregated).limit(limit).as_scalar()
+        return subquery.query.add_column(aggregated).limit(limit).label(None)
```

Comment:
We could have applied this .label(None) [here](https://github.com/specify/specify7/blob/f4a107d3d136d69cf45f3309aec450f58266dd9b/specifyweb/stored_queries/execution.py#L625-L627) too, but I think it is worth handling it at the level of aggregate() if someplace might also use it in the future